iree/compiler/Dialect/Flow/Transforms/DispatchConfig.cpp

```diff
@@ -21,13 +21,35 @@
 #include "mlir/Dialect/StandardOps/IR/Ops.h"
 #include "tensorflow/compiler/mlir/xla/ir/hlo_ops.h"
 
-#define DEBUG_TYPE "iree-dispatch-config"
+#define DEBUG_TYPE "xxx"
```

Comment:
intentional?

Reply:
fixed in an update that hasn't pushed for some reason.